src/main/java/TextView.java

```diff
@@ -20,9 +20,13 @@
 	public void update(Observable o, Object arg)
 	{
 		Model model = (Model) o;
+		
+		System.out.println("");
+		System.out.println("LOWERCASE: BLACK | UPPERCASE: WHITE ");
+		System.out.println(model.getWhosTurn() + "'S Turn ");
+		System.out.println("Turn Number: " +model.getTurnCount());
+		System.out.println("");
 		drawBoard(model.getBoard());
-
-		System.out.println("The view has called refresh");
 	}
 
 	/**
```

Comment:
- [ ] Specify in the comment, that you are also printing other game info (who's turn, number of turns, etc...)